skorch/tests/test_classifier.py

```diff
@@ -208,3 +208,27 @@ def test_target_2d_raises(self, net, data):
 
         assert exc.value.args[0] == (
             "The target data should be 1-dimensional.")
+
+    def test_custom_loss_does_not_call_sigmoid(
+            self, net_cls, data, module_cls, monkeypatch):
+        mock = Mock(side_effect=lambda x: x)
+        monkeypatch.setattr(torch.nn.functional, "sigmoid", mock)
+
+        net = net_cls(module_cls, max_epochs=20, lr=0.1, criterion=nn.MSELoss)
```

Comment:
Do we need 20 epochs here? Probably not a huge loss of time, but there seems to be no benefit.